### ISSUE TITLE
Compile error - missing gtk.h

### ISSUE THREAD

**Issue by joco-nz** [grotius-cnc/linuxcnc#8]
Compile dependency issue.  Environment is configured and proven to compile standard linuxcnc from master.
Output from cmake attached.

Issue: error with missing gtk.h

[cmake_out.txt](https://github.com/grotius-cnc/linuxcnc/files/13245499/cmake_out.txt)

**Comment by grotius-cnc** [comment]
https://askubuntu.com/questions/397432/fatal-error-gtk-gtk-h-no-such-file-or-directory-using-make

In our cmake we used the gtk2 directory  "usr/include/gtk-2.0" to compile classicladder for example.

I suggest this will sove it:

$ sudo apt-get install libgtk2.0-dev

**Comment by joco-nz** [comment]
That worked for the build.  Suggest adding that to the dependency part of the make script like the other lib.

The build script finishes trying to run the axis_mm.ini sim.  However that fails.   I manually sourced the rip_environment script but also no joy.

[tail_good_build.txt](https://github.com/grotius-cnc/linuxcnc/files/13247281/tail_good_build.txt)

[trying_to_run.txt](https://github.com/grotius-cnc/linuxcnc/files/13247282/trying_to_run.txt)

**Comment by grotius-cnc** [comment]
Can you try to build linuxcnc the original way and start a axis gui?
This would help in problem solving.

**Comment by joco-nz** [comment]
okay tested.  This is what I did and what I found.
1. went did a normal linuxcnc compile from src dir.  Got the autogen and configure setup and the started with 'make clean' then 'make -j3'.
2. source ../scripts/rip-environment
3. run linuxcnc and chose axis mm.   Works!

4. make clean.   cd to cmake.  build_all.   Buidls script tries to start axis, fails.

5. cd back to src.  make clean.  make - j3
6. cd back to cmake.  build_all.  Build completes and script successfully runs axis with new s-curve module.

7. cd back to src.  make clean.
8. cd back to cmake.  build_all (fails to run axis as expected). Then manually run linuxcnc from command line and select axis_mm.  Error with text attached.
[error.txt](https://github.com/grotius-cnc/linuxcnc/files/13270822/error.txt)
